style.css

```diff
@@ -1,340 +1,1391 @@
+    background-color: #0265DC;
+    color: #F8F7FF;
+}
+
+/* Blocks for all pages */
+
+.main {
+    max-width: 1630px;
+    margin: 0 auto;
+    padding: 0 15px;
+}
+
+.btn-wrapper {
+    display: flex;
+    justify-content: center;
+}
+
+.btn {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: max-content;
+    height: 48px;
+    padding: 0 22px;
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 28px;
+    text-decoration: none;
+    border: 2px solid #D0D7FB2B;
+    color: var(--color-text);
+    box-sizing: border-box;
+}
+
+.btn--icon {
+    padding: 0 10px 0 48px;
+}
+
+.btn--icon img {
+    position: absolute;
+    top: 10px;
+    left: 10px;
+}
+
+.btn--color {
+    background-color: var(--color-blue);
+    border-color: var(--color-blue);
+}
+
+.btn--full {
+    width: 100%;
+    text-align: center;
+}
+
+.btn--white {
+    background-color: #fff;
+}
+
+.image-label {
+    position: absolute;
+    top: 5px;
+    right: 3px;
+    padding: 0 5px;
+    background-color: #fff;
+    color: var(--color-blue);
+}
+
+.image-label--sold {
+    background-color: #EA3829;
+    color: #F5F8FB;
+}
+
+.subtitle {
+    margin-bottom: 40px;
+    font-family: var(--font-secondary);
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+    text-align: center;
+    color: #F8F7FF;
+}
+
+/* Header all pages */
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1758px;
+    margin: 0 auto;
+    padding: 32px 15px;
+}
+
+.header-nav__wrapper {
+    display: flex;
+}
+
+.header-nav__item a {
+    display: block;
+    border-bottom: 2px solid #0D0D15;
+    padding: 6px 20px;
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 24px;
+}
+
+.header-nav__item a:hover {
+
+}
+
+.header-actions {
+    display: flex;
+    margin-left: 20px;
+}
+
+.search-form {
+    position: relative;
+    padding-right: 32px;
+}
+
+.search-form:after {
+    content: '';
+    position: absolute;
+    top: 10px;
+    right: 15px;
+    width: 2px;
+    height: 24px;
+    background-color: #D0D7FB2B;
+}
+
+.search-form__input {
+    width: 188px;
+    height: 48px;
+    padding: 0 0 0 48px;
+    border: 2px solid rgba(208, 215, 251, 0.17);
+    font-size: 14px;
+    line-height: 20px;
+    color: var(--color-text);
+    background-color: transparent;
+    box-sizing: border-box;
+}
+
+.search-form__btn {
+    position: absolute;
+    left: 12px;
+    top: 12px;
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+}
+
+/* Index page Intro block */
+
+.intro {
+    background: radial-gradient(42.5% 42.5% at 22.28% -4.22%, rgba(2, 101, 220, 0.5) 0%, rgba(2, 101, 220, 0) 100%);
+    margin-bottom: 40px;
+}
+
+.intro__wrapper {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    max-width: 1630px;
+    margin: 0 auto;
+    padding: 0 15px;
+}
+
+.intro__wrapper::after {
+    content: url('images/bg-2.webp');
```

Comment:
This is should be background-image. Please change.

Reply:
fix in all places